ui/src/app/views/resource-card/component.tsx

```diff
@@ -15,6 +20,84 @@ import {environment} from 'environments/environment';
 import {ConfirmDeleteModalComponent} from 'app/views/confirm-delete-modal/component';
 import {EditModalComponent} from 'app/views/edit-modal/component';
 
+const MenuItem = ({ icon, children, ...props }) => {
```

Comment:
Should MenuItem be in a more central location, or is the styling specific to the resource menu? We have at least one other such menu in the application (workspace-nav-bar dropdown menu).

Reply:
It will probably get used again, at which point we'll move it to a more central location. I tend to delay that decision until we have two concrete examples, because it's much easier to get the abstraction right.

Reply:
Sounds good.